modules/gcp-velero/gcs.tf

```diff
@@ -5,7 +5,7 @@
  */
 
 # Create a unique GCS bucket per cluster
-resource "google_storage_bucket" "main" {
```

Comment:
I suggest maintaining the original name, changing it will require recreating the bucket if you update from a previous version.

Reply:
✅  thanks